api-spec/v1/transactions-api.yaml

```diff
@@ -742,6 +742,7 @@ components:
         - ACTIVATED
         - AUTHORIZATION_REQUESTED
         - AUTHORIZATION_COMPLETED
+        - CLOSURE_REQUESTED
```

Comment:
TODO: remember to update this specs also on infra

Reply:
https://github.com/pagopa/pagopa-infra/pull/1686